src/patch-builtins.js

```diff
@@ -177,6 +179,11 @@ let elementMixin = utils.extendAll({
    */
   get assignedSlot() {
     return getAssignedSlot(this);
+  },
+
+  dispatchEvent(event) {
```

Comment:
Technically, `dispatchEvent` also exists on e.g. text nodes, not just elements.  Rather cornery though.